backend/src/api/routes/ai/index.routes.ts

```
@@ -6,30 +6,19 @@ import { EmbeddingService } from '@/services/ai/embedding.service.js';
 import logger from '@/utils/logger.js';
 import {
-  createAIConfigurationRequestSchema,
-  updateAIConfigurationRequestSchema,
-  getAIUsageRequestSchema,
-  getAIUsageSummaryRequestSchema,
   chatCompletionRequestSchema,
   imageGenerationRequestSchema,
   embeddingsRequestSchema,
-  setGatewayBYOKKeyRequestSchema,
 } from '@insforge/shared-schemas';
+import type { AIOverviewRange } from '@insforge/shared-schemas';
```

ESLint

[error] '@insforge/shared-schemas' import is duplicated.

(no-duplicate-imports)